scripts/ios/get_ios_certs.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 
+export KEYCHAIN_NAME=atb
```

Comment:
How does other OMS use this value @jorelosorio ?

Reply:
this command is in the dev machine for this case, but for the github actions, it uses CI as a name.

Reply:
What about when running up dev app for other transport companies? Does it still work with atb as keychain name?